src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  const object = {
+    result: 0,
+
+    operate(callback, number) {
+      callback(this, number);
+
+      return this;
+    },
+
+    add(obj, number) {
+      obj.result += number;
+    },
```

Comment:
```suggestion
    add(number) {
      this.result += number;
      
      return this;
    },
```
same for subtract, multiply and divide methods.